.devcontainer/Dockerfile

```
@@ -1,13 +1,19 @@
 FROM python:3.12-slim-bullseye as builder
 
+
+ARG DOPPLER_TOKEN
+ARG DOPPLER_PROJECT
+ARG DOPPLER_CONFIG
+
+FROM python:3.12-slim-bookworm as builder
+
 RUN apt-get update && apt-get install -y --no-install-recommends apt-transport-https ca-certificates libmagic1 curl libenchant-2-dev gnupg make git && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)